python/lsst/ts/watcher/model.py

```diff
@@ -230,6 +230,37 @@ def get_rules(self, name_regex):
         compiled_re = re.compile(name_regex)
         return [rule for name, rule in self.rules.items() if compiled_re.match(name) is not None]
 
+    def mute_alarm(self, name, duration, severity, user):
+        """Mute one or more alarms for a specified duration.
+
+        Parameters
+        ----------
+        name : `str`
+            Alarm name.
+            If a regular expression then acknowledge all matching alarms.
+        duration : `float`
+            How long to mute the alarm (sec)
+        severity : `lsst.ts.idl.enums.Watcher.AlarmSeverity` or `int`
+            Severity to acknowledge. If the severity goes above
+            this level the alarm will unacknowledge itself.
```

Comment:
unacknowledge, or unmute?

Reply:
This was a copy/paste error that I have fixed. If the level goes above severity nothing happens to the mute condition of the alarm, but the UI should display the alarm normally.